tests/unit/importer/test_importer.py

```diff
@@ -138,33 +147,118 @@ def test_import_file__when_generate_json_has_errors__then_report_errors(self):
         self.assertEqual(self.mock_ingest_api.create_submission_error.call_count, 2)
 
     @patch('ingest.importer.submission.ingest_submitter.IngestSubmitter.link_submission_to_project')
-    def test_import_file__when_generate_json_has_no_project__then__do_not_link_to_project(self, mock_link_submission_to_project):
+    def test_import_file__when_spreadsheet_has_no_project_reference__then_do_not_link_to_project(self, mock_link_to_project):
```

Comment:
That's probably the longest function name I have ever seen :) would probably violate pep style guideline?

Reply:
I've extracted the importer.import_file tests to a different test suite file so that I can shorten the tests function names. It would be nice if nesting of test cases can be done in python's unittests syntax. This should avoid long test case function names.

Reply:
We could group tests to modules & suites so that the suite's name conveys some of the information.
In this case, for example, the module is called `test_importer`, so the prefix `test_import_` in the function name is redendant.